regression/iterate_data.py

```diff
@@ -104,10 +109,9 @@ def main(args: Namespace, kwargs: dict[str, str]) -> None:
     iter_time = []
     start_time = 0
     for epoch in range(args.epochs):
-        if epoch > 0:
-            start_time = time.time()
+        start_time = time.time()
         for batch in dataloader:
-            if args.sample_order_file is not None:
+            if args.sample_order_file is not None and epoch == 0:
```

Comment:
curious why we're not writing samples to the file past epoch 0?

Reply:
Good point. I will have to change the logic in future for testing different sampling_method